zeus/carbon/electricity_maps_carbon_intensity_provider.py

```diff
@@ -0,0 +1,30 @@
+"""Carbon Intensity Provider using ElectrictyMaps API."""
+from zeus.carbon.carbon_intensity_provider import CarbonIntensityProvider
+import requests
+
+
+class ElectricityMapsCarbonIntensityProvider(CarbonIntensityProvider):
```

Comment:
Do they have API rate limits for people who don't pay?

Reply:
They don't explicitly describe the rate limit. [This](https://www.electricitymaps.com/pricing) says both individual and enterprise use dont have "increase rate limits". I looked into the [commercial api](https://www.electricitymaps.com/zonal-api-access) and it says the rate is 500k calls / month.

I think this should more than enough, as it seems that the individual use has the same rate limit as the commercial use.